agent/agent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/TracerInstaller.java

```diff
@@ -40,8 +56,29 @@ public static void installAgentTracer() {
                             .build());
         }
         // if changing the span processor to something async, flush it in the shutdown hook before flushing TelemetryClient
-        OpenTelemetrySdk.getTracerManagement()
-                .addSpanProcessor(SimpleSpanProcessor.newBuilder(new Exporter(telemetryClient)).build());
+        if (!processors.isEmpty()) {
+            SpanExporter prevExporter = null;
```

Comment:
name this `currExporter` then don't need both, and keep reassigning new value to currExporter